### ISSUE TITLE
material icons - two-tone icons not showing - in chrome desktop

### ISSUE THREAD

**Issue by JustCodin** [google/material-design-icons#1831]
Material icons two-tone font was previously working fine, but now not visible on chrome desktop, although visible on chrome mobile and other browsers (safari, firefox, etc). Although, regular material icons are visible.

I.e. load font, and use it. Regular works (class="material-icons"), but not two-tone (class="material-icons-two-tone"). Simple example below to test locally.

<!DOCTYPE html>
<html lang="en">
<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Material+Icons|Material+Icons+Outlined|Material+Icons+Two+Tone|Material+Icons+Round|Material+Icons+Sharp">
<body>
<span class="material-icons-two-tone">assistant</span>
</body>
</html>

**Comment by JustCodin** [comment]
From limited testing (2 macs, 2 iphones, 1 windows), it seems like this issue in only occurring on Mac chrome desktop. The icon does not appear in incognito.

**Comment by tphinney** [comment]
So, it is only in Mac Chrome desktop, and in incognito mode?

Probably makes sense to report it to the Chrome team, then.

The icons themselves are not under active development, as Google is focused on Material Symbols now. (They do not have a two-tone version, although they do have cool variable font capability and come in sharp and rounded versions.)

**Comment by JustCodin** [comment]
No, it’s in all of chrome in Mac desktop. Just wanted to call out an extra testing step that it’s also in incognito. 

Thanks for the context. I did see that update, but it shouldn’t break existing behavior, right? I’ve also reported it to the chrome team via feedback in the browser. Any other recommended channels to report this issue?

**Comment by tphinney** [comment]
Ah, thanks for the clarification.

And correct: I was not suggesting that it was OK that something broke with a Chrome update! AFAIK that is the recommended reporting channel (see for example https://support.google.com/chrome/answer/95315?hl=en&co=GENIE.Platform%3DDesktop)
